### ISSUE TITLE
Thermostat attribute undefined

### ISSUE THREAD

**Issue by dejongac** [jimz011/homekit-infused#198]
I see a difference in behaviour of the thermostat attribute: state.

When i create a view with Thermostat add i get the correct state

  Klimaat:
    icon: mdi:home-thermometer-outline
    show_in_navbar: true
    show_in_favorites: false
    addons:
      thermostat:
        - title: Thermostaten Begane grond
          entities:
            - entity: climate.woonkamer
              attribute: state
              box_shadow: fancy
   
When i use it as an entity with the Buttons addon i get undefined.

   addons:
      button: 
        - title: Woonkamer    
          icon: mdi:sofa
          entities:
            - entity: climate.woonkamer
              attribute: state
              box_shadow: fancy

I it correct there is a difference?

regards,

Albert

**Comment by jimz011** [comment]
Yes I had noticed and fixed it a while back, it will be available in the upcoming update (expect Friday or Saturday).

**Comment by dejongac** [comment]
Thanks for the quick reply and solution.
